src/ontology/mondo-edit.obo

```diff
@@ -487910,16 +487910,16 @@ subset: rare
 synonym: "C282Y/C282Y hemochromatosis" EXACT [EFO:0006513]
 synonym: "classic hemochromatosis" EXACT [EFO:0006513]
 synonym: "hemochromatosis" EXACT [OMIM:235200]
-synonym: "hemochromatosis type 1" EXACT [DOID:0111029] {OMO:0002001="https://w3id.org/information-resource-registry/clingen"}
+synonym: "hemochromatosis type 1" EXACT [OMIM:235200] {OMO:0002001="https://w3id.org/information-resource-registry/clingen"}
 synonym: "hemochromatosis, type 1" EXACT [OMIM:235200]
 synonym: "hfe hemochromatosis, modifier of" EXACT []
 synonym: "HFE-associated hereditary hemochromatosis" EXACT [EFO:0006513, NCIT:C84764]
 synonym: "HFE-related hemochromatosis" EXACT [EFO:0006513]
-synonym: "HFE1" EXACT ABBREVIATION [DOID:0111029, MONDO:Lexical, OMIM:235200]
+synonym: "HFE1" EXACT ABBREVIATION [MONDO:Lexical, OMIM:235200]
 synonym: "symptomatic form of classic hemochromatosis" NARROW []
-synonym: "symptomatic form of hemochromatosis type 1" EXACT [DOID:0111029]
+synonym: "symptomatic form of hemochromatosis type 1" NARROW [Orphanet:465508]
 synonym: "symptomatic form of HFE-related hereditary hemochromatosis" NARROW []
-xref: DOID:0111029 {source="MONDO:equivalentTo"}
+xref: DOID:0111029 {source="MONDO:mondoIsBroaderThanSource"}
```

Comment:
@MeeSiing 
I don't think this is correct. The synonym was broader, but DOID seems to be equivalent. Isn't it?